.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: python
 python:
   - "2.7"
+  - "3.6"
+  - "3.7"
+  - "3.8"
```

Comment:
I guess it doesn't hurt to support more if it's easy enough, but really we'd only need 3.7, cause thats what the vfx reference platform recommends anyway: https://vfxplatform.com/

Reply:
It doesn't hurt to ensure that it works with future version. As long as the current standard version is in the tests..

Reply:
3.6 removed